tests/fixtures/snapshot_date.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from typing import Any
+
+import pytest
+from syrupy.matchers import path_type
+
+
+def validate_time(timestamp: datetime.datetime | Any):
```

Comment:
You may be able to avoid the extra `lambda` below with:
```suggestion
def validate_time(timestamp: datetime.datetime | Any, _):
```

Reply:
Had to default set it to None